pitch.md

```diff
@@ -57,13 +125,38 @@ the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+This project is pretty large. I feel like there is a huge amount of work to be
+done in language design. Figuring out how to integrate lots of different methods
+of robot control and allow users to easily specify logic without limiting them
+sounds like a difficult challenge. Implementation will probably also require a
+bit of work, but I would hazard that about 80\% of this project is just figuring
+out what is good for the users. The language has to be as big as a GPL, but this
+is easy if the DSL is internal.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This is a pretty big project. I'm hoping to make something that will be
+personally useful to me and potentially useful to others doing robotics. As I
+mentioned before, there are lots of other languages that try to do similar
+things to what I'm doing, and it will probably take lots of work to figure out a
+solution that I find better than theirs. If this DSL ends up being external,
```

Comment:
Correct me if I'm wrong, but it sounds like you want to simplify state-centric logic specifications. I think it's worth making a mental note that this goal should probably be your primary concern, and you should be wary of adding features in your language in response to shortcomings in these other languages, because the shortcomings might fall outside of your personal goal.